### DISCUSSION TITLE
Batch change the same paratmer of a group of parts

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#8113] [Q&A] [answered]
It turns out to be painful that if I have to change a parameter of a group of parts, I have to change them individually, is there a way to change them naturally just like drag and pull in excel as table?

**Comment by matmair** [comment, marked as answer]
No, InvenTree is not Excel. We do not have the financial or personal resources to provide this level of interaction

**Reply by jesseyu1984918** [reply]
How about batch add a paramter for all parts under one category? One thing I find tedious is that if I need to add one new parameter(not value) to all parts under one category, I have to add them one by one, this is a bit of tedious

**Reply by SchrodingersGat** [reply]
@jesseyu1984918 you are welcome to contribute some user interface code to manage this! The new user interface (in active development) is being developed in react, and provides much more powerful functionality. We do not have a priority for this particular feature right now but would not reject it if you can provide a good implementation
